### DISCUSSION TITLE
登陆webdav成功，但加载不了数据，提示：null check operator used on a null value

### DISCUSSION THREAD

**Discussion by mousekane** [GhostenEditor/Ghosten-Player#5] [General]
登陆webdav成功，但加载不了数据，提示：null check operator used on a null value

**Comment by GhostenEditor** [comment]
是用的alist吗，alist要加/dav

**Reply by mousekane** [reply]
感谢回复，
经测试，本地alist服务器加/dav，加载成功。
远程服务器加/dav仍然无法加载（网址:端口/dav）；
另外alist挂载到阿里网盘，只能读取mp4文件，同文件夹的rmvb无法读取

**Reply by GhostenEditor** [reply]
按理来说地址是通的就能连上，和本地还是远程服务器无关。我感觉还是地址的问题，因为我不知道你的远程服务器是如何代理的，所以无法定位问题。准确的说alist的webdav的地址应该是 http(s)://domain:port/[path]/dav, 如果你可以通过/dav前面的那一段地址访问到alist的web页面，那应该就可以连上了。
目前rmvb文件确实被过滤掉了，会在下个版本修复。
感谢反馈
